indexer/README.md

```diff
@@ -143,15 +155,23 @@ Click on link. Your web browser should appear requesting you to login. Login wit
 1. Copy down the uuid for the subscription.
 
 ## Reindexing
-
-Run the command below.
-```
-python test/find-golden-tickets.py --dss-url https://<your DSS URL> --indexer-url https://<your elasticsearch endpoint> --es-query {}`
-```
+In cases where the Azul index in Elasticsearch may have become inconsistent with the data stored in the DSS, reindexing of the DSS content
+to the Elasticsearch Azul index may be performed as follows:
+1. If the DSS to be reindexed requires authentication/authorization for the for access to the `GET /bundles`, `GET /files` or `POST /search` API operations,
+   then set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` as described in step 3 above.
+   When reindexing, a fully qualified path to the `gcp-credentials.json` file is should be used.
+   For example:
+   ```
+   export GOOGLE_APPLICATION_CREDENTIALS=<path to project>/chalicelib/gcp-credentials.json
+   ```
+2. Run the command below.
+    ```
+    python test/find-golden-tickets.py --dss-url https://<your DSS URL> --indexer-url <azul-indexer-url> --es-query "{}"
+    ```
 where you replace the `<>` value with your own values.
-- `<your DSS URL>` is the endpoint of the HCA DSS and the /v1 url path. (e.g. example-dss.ucsc-cgp.org/v1)
-- `<your elasticsearch endpoint>` is url of Azul's Elasticsearch Domain. (e.g. search-example-lambda-abcdefghijklmnopqrstuvwxyz.us-west-2.es.amazonaws.com)
-       
+- `<your DSS URL>` is the endpoint of the HCA DSS plus the Swagger base path (e.g. `/v1`, as in `example-dss.ucsc-cgp.org/v1`)
+- `<azul-indexer-url>`: your azul indexer's url. If you need to know what is your url, following the directions 
```

Comment:
I would replace this with

    If you need to know what your URL is, follow the instructions in [Finding the URL of your Azul Indexer Lambda](#finding-the-url-of-your-azul-indexer-lambda).

Reply:
Done